pyecmech/ecmech_classes.hpp

```diff
@@ -412,16 +414,25 @@ class pyEvptn_norm : public pyevptn_base
 
 };
 
+typedef ecmech::KineticsKMBalD<true, false, false, false, 1> Kin_KMBalD_TFF;
```

Comment:
Why add all of these typedefs here rather than using them from ECMech_cases.h?

Reply:
So, I unfortunately can't use them when the library is built with cuda or else we get a number of build errors when the final python file is being built :( 

Now, if we split up the case file into two where one had all the non-cuda/gpu specific typedefs in, and then the other file imported that one and had cuda/gpu type things we could make use of the case file. However, the current issue is the raja foralls in evptnWrap are depend on a few raja defines which know whether or not things should be compiled with cuda or not :/

Reply:
Hmm... I do not follow the details, but that is okay. I see how that you had mentioned this code duplication issue elsewhere -- I was triaging things in somewhat haphazard fashion today. Maybe just add a comment about the undesirable code duplication and the motivation for it. Something to help people (or yourself later down the line) who come across this and might be confused.

Reply:
I think I've figured out a way to avoid the code duplication and avoid the build failing during the last portion of the python module creation, but I won't get to it until Friday.